src/SegmentedControl/SegmentedControl.jsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import lodash from "lodash";
+
+import "./SegmentedControl.less";
+
+/*
+  SegmentedControl selection. Shows several options
+  as defined in the props and allows the user to select
+  one of those options.
+*/
+export class SegmentedControl extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {selected: props.defaultValue};
+    this.onSelect = this.onSelect.bind(this);
+  }
+
+  onSelect(e) {
+    const value = e.target.getAttribute("value");
+    this.setState({selected: value});
+    if (this.props.onSelect !== undefined) {
```

Comment:
safer to use `if (this.props.onSelect) {...}` since `this.props.onSelect = null` would pass this check